.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
+  - 7.2
```

Comment:
Any particular reason for skipping 7.0 and 7.1?

Reply:
They are both out of support.

Reply:
Yeah. we can drop it totally next year.